### ISSUE TITLE
Can't get locast2plex to work after starting successfully once.

### ISSUE THREAD

**Issue by TritiumAE** [tgorgdotcom/locast2plex#112]
I'm not seeing a place where logs would go for locast2plex to see what might be going on.
Windows 10
python3
V.5
PlexMediaServer-1.16.0.1226

I can see port 1900  Windows Host:  SSDP Discovery
But no errors related to why locast2plex isn't loading.

Any help would be appreciated.
I was able to get it running and added to the plex server once, but the latest version seems to break things, so I reverted back to the version you see above.

Thanks for any help.

**Comment by tgorgdotcom** [comment]
Hmmm, the prior version uses python2, not sure if that's the cause...

**Comment by TritiumAE** [comment]
> Hmmm, the prior version uses python2, not sure if that's the cause...

Yeah,  I believe the site recommended version 3, so I installed that one.

**Comment by TritiumAE** [comment]
So, I'm not sure what the deal was.  I thought I had python in the path.  When I ran the command, it worked once.. but for some reason it provide any output, which is crazy because even though zero output, it ran the first time, but wouldn't run after.

In the end, I installed python3 from the Microsoft Store, and that seems to have corrected the issue for me.  So most likely human error on my part.
